sentry-android-ndk/src/main/jni/sentry.c

```diff
@@ -398,5 +398,5 @@ Java_io_sentry_android_ndk_NativeModuleListLoader_nativeLoadModuleList(JNIEnv *e
 
 JNIEXPORT void JNICALL
 Java_io_sentry_android_ndk_SentryNdk_shutdown(JNIEnv *env, jclass cls) {
```

Comment:
might as well rename this too

Reply:
I cant because the java method is called `close`, but I need to map the same method (JNI) within the same class.
users call `Sentry.close()` only, not `shutdown`